token/sdk/sdk.go

```diff
@@ -205,9 +179,18 @@ func (p *SDK) Start(ctx context.Context) error {
 		logger.Infof("start token management service [%s]...", tmsID)
 
 		// connect network
-		if err := p.postInitializer.ConnectNetwork(tmsID.Network, tmsID.Channel, tmsID.Namespace); err != nil {
+		net := network.GetInstance(p.registry, tmsID.Network, tmsID.Channel)
```

Comment:
Since we are in the sdk and we have access to the network provider, the tms provider etc, maybe we could use those instead of calling the registry

Reply:
👍